server/controllers/articleController.js

```diff
@@ -35,6 +37,9 @@ export default class ArticleController {
             });
           });
         })
+          .then(() => {
+            Notification.notifyForArticle(req.user.id, articleSlug, title, req.user.username);
```

Comment:
This method called here sends a notification to all the followers of the  Author.